hoomd/logging.py

```diff
@@ -378,6 +383,8 @@ def log(func=None, *, is_property=True, category='scalar', default=True):
             quantities even when logging other quantities of that type. The
             default category allows for these to be pass over by
             `hoomd.logging.Logger` objects by default. Argument keyword only.
+        requires_attach (`bool`, optional): Whether this property is
+            accessible before attaching.
```

Comment:
The name of the argument seems to be the opposite of its own description (if a property requires attachment then it _is not_ accessible before attaching)? Change description to "Whether this property requires attachment before access"?